{"name": "test_query_order", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <scripts.testOrder.TestOrderApi object at 0x000002C1A26C4CF8>\n\n    def test_query_order(self):\n        \"\"\"查询订单\"\"\"\n        # 订单id\n        order_id = 115\n        # 响应对象\n        res = ApiFactory.get_order_api().query_order_api(order_id)\n    \n        # 断言 状态码\n>       utils.common_assert_code(res)\n\nscripts\\testOrder.py:54: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nres = <Response [401]>, code = 200\n\n    def common_assert_code(res, code=200):\n        \"\"\"\n        通用断言状态码\n        :param res: 响应对象\n        :param code200: 状态码 默认值200\n        :return:\n        \"\"\"\n>       assert res.status_code == code\nE       AssertionError\n\nutils.py:8: AssertionError"}, "description": "查询订单", "attachments": [{"name": "log", "source": "cbb42240-bd23-4139-9976-a32e02bf2722-attachment.txt", "type": "text/plain"}], "start": 1602559728685, "stop": 1602559728707, "uuid": "6c2e5c70-2cb2-4183-abb4-f0f56b574ae5", "historyId": "1c49826f8ac9a63523112cb6b3894d94", "testCaseId": "03f150f83a542246dd0eac87d34a62ce", "fullName": "scripts.testOrder.TestOrderApi#test_query_order", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "testOrder"}, {"name": "subSuite", "value": "TestOrderApi"}, {"name": "host", "value": "DESKTOP-7793Q7A"}, {"name": "thread", "value": "19652-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.testOrder"}]}